protocol/x/clob/e2e/long_term_orders_test.go

```diff
@@ -1331,6 +1333,7 @@ func TestPlaceLongTermOrder(t *testing.T) {
 											25_000_000,
 											LongTermPlaceOrder_Alice_Num0_Id0_Clob0_Buy2_Price50000_GTBT5.Order.GetBaseQuantums(),
 											PlaceOrder_Bob_Num0_Id1_Clob0_Sell1_Price50000_GTB20.Order.GetBaseQuantums(),
+											big.NewInt(0),
```

Comment:
Let's cover the full path in some e2e test https://linear.app/dydx/issue/OTE-730/add-e2e-tests-for-affiliates. Might make sense to add to some of these existing tests instead of creating entirely new ones